### PR TITLE
Updated Chat with Video Project

### PR DESCRIPTION
**Pull request by movchan74**
**Summary:**
This PR updates the chat with video project, introduces a new LLM with larger context size, and makes adjustments to dependencies and error handling.

**Key Changes:**
1. Removed timeline trimming from the chat with video project.
2. Added InternLM2.5 7B chat model for use in the chat with video project.
3. Implemented VideoTooLongException for videos exceeding 20 minutes.
4. Disabled batched Whisper functionality from our fork temporarily.
5. Replaced opencv-python with opencv-python-headless to remove libgl dependency.

### CODE REVIEW

**Review by evanderiel — APPROVED** [review]
Looks good

**Review by HRashidi — APPROVED** [review]
👍🏽 
question: What do we want to do about `transcribe_in_chunks`? keeping them as commented for later?

File: aana/exceptions/io.py [hunk 1]
```diff
@@ -102,3 +102,30 @@ class VideoReadingException(VideoException):
...
+        max_len (float): the maximum allowed length of the video in seconds
+    """
+
+    def __init__(self, video: "Video", video_len: float, max_len: float):
+        """Initialize the exception.
+
+        Args:
+            video (Video): the video that caused the exception
+            video_len (float): the length of the video in seconds
+            max_len (float): the maximum allowed length of the video in seconds
```
Comment (ashwinnair14) [inline aana/exceptions/io.py:122]: DO we need this twice?

**Comment by movchan74** [comment]
> 👍🏽 question: What do we want to do about `transcribe_in_chunks`? keeping them as commented for later?

Keep it for later. Update the code once batched faster whisper is merged into a main repo.

File: aana/projects/chat_with_video/endpoints.py [hunk 2]
```diff
@@ -162,16 +176,16 @@ async def run(
             update_video_status(media_id=media_id, status=Status.RUNNING)
             audio: Audio = extract_audio(video=video_obj)
 
-            vad_output = await self.vad_handle.asr_preprocess_vad(
-                audio=audio, params=vad_params
-            )
-            vad_segments = vad_output["segments"]
+            # vad_output = await self.vad_handle.asr_preprocess_vad(
```
Comment (ashwinnair14) [inline aana/projects/chat_with_video/endpoints.py:179]: Maybe add a comment sayin we are keeing this in for batch whisper changes?

File: aana/exceptions/io.py (hunk 1 above)
Comment (movchan74) [reply aana/exceptions/io.py:122]: Do you mean Attributes and Args? Not sure but I keep it consistent with the exceptions we had before.

**Review by ashwinnair14 — APPROVED** [review]
Just one comment about the commenting of the batched whisper. 

Leaving a TODO or a comment like batched whisper would make it easier

File: aana/projects/chat_with_video/endpoints.py (hunk 2 above)
Comment (movchan74) [reply aana/projects/chat_with_video/endpoints.py:180]: Done

**Review by ashwinnair14 — APPROVED** [review]
LGTM

**Pull request merged**